pages/golfers/[id].js

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react'
+import { useRouter } from 'next/router'
+import Layout from '../../components/Layout'
+import ScoreCard from '../../components/ScoreCard'
+import useUserScores from '../../lib/useUserScores'
+
+const ProfilePage = () => {
+  const router = useRouter()
+  const { id } = router.query
+  const { scores, error, userName } = useUserScores(id)
+
+  return <Layout>
+    <>
+      {error ? (
+        error
+      ) : (
+        <>
+          {userName && <h1>Scores for {userName} : </h1>}
```

Comment:
This will display `Scores for George Marcus :` 
Let's remove the space before `:`